healthcare/api-client/v1/fhir/fhir_resources_test.py

```diff
@@ -40,23 +42,35 @@
 
 BACKOFF_MAX_TIME = 750
 
+client = discovery.build("healthcare", "v1")
+
+
+class OperationNotComplete(Exception):
+    """Operation is not yet complete"""
+
+    pass
+
+
+@retry.Retry(predicate=retry.if_exception_type(OperationNotComplete))
```

Comment:
The default retry deadline is [2 minutes (120 seconds)](https://github.com/googleapis/python-api-core/blob/4422ccedd46fb1fd1e99844381681d8aa314c90c/google/api_core/retry.py#L75). It seemed good enough for the CI run on this PR and my local runs. If this flakes we can bump it higher.